src/app/features/collectibles/types.ts

```diff
@@ -0,0 +1,20 @@
+import * as yup from 'yup';
```

Comment:
Filename is kind of confusing. No types declared here, only validation. `ordinal-validator` or similar?